Source/WebCore/platform/network/NetworkStorageSession.h

```diff
@@ -145,6 +145,10 @@ class NetworkStorageSession : public CanMakeWeakPtr<NetworkStorageSession> {
     WEBCORE_EXPORT void setCookiesFromHTTPResponse(const URL& firstParty, const URL&, const String&) const;
     WEBCORE_EXPORT void setCookieAcceptPolicy(CookieAcceptPolicy) const;
     WEBCORE_EXPORT void setProxySettings(const CurlProxySettings&);
+
+    WEBCORE_EXPORT void setAlternativeServicesStorageFile(const String& cacheFile) const;
```

Comment:
Can we put this in the constructor instead?

Reply:
Fixed